epic.go

```diff
@@ -166,13 +195,13 @@ func (s *Session) QueryPlayerById(accountId string) (*statsResponse, error) {
 	req.Header.Set("Authorization", fmt.Sprintf("%v %v", AuthBearer, s.AccessToken))
 
 	sr := &statsResponse{}
-	resp, err := s.client.Do(req, sr)
+	resp, _, err := s.client.Do(req, sr)
```

Comment:
Why add the new return variable if it's just thrown away? I'd prefer the `X-Epic-Device-ID` to be built into the error with `fmt.Errorf()`. What is this device ID for? An identifier of you from them?